stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/ClientAndUsers.java

```diff
@@ -151,4 +158,73 @@ public void onError(@NotNull ChatError error) {
             }
         });
     }
+
+    public void queryingUsersById() {
```

Comment:
I was a bit confused that these querying related snippets ended up on this PR too, as it just says "Events" everywhere - but they look correct 😄

Reply:
Sorry, I have updated PR's title and description 🙈

Reply:
No worries, thanks 😄